pkgs/development/python-modules/pyside/default.nix

```diff
@@ -1,14 +1,15 @@
-{ lib, fetchurl, cmake, buildPythonPackage, pysideGeneratorrunner, pysideShiboken, qt4, mesa, libGL }:
+{ lib, fetchFromGitHub, cmake, buildPythonPackage, pysideGeneratorrunner, pysideShiboken, qt4, mesa, libGL }:
 
-# This derivation provides a Python module and should therefore be called via `python-packages.nix`.
 buildPythonPackage rec {
   pname = "pyside";
   version = "1.2.4";
   format = "other";
 
-  src = fetchurl {
-    url = "https://github.com/PySide/PySide/archive/${version}.tar.gz";
-    sha256 = "90f2d736e2192ac69e5a2ac798fce2b5f7bf179269daa2ec262986d488c3b0f7";
+  src = fetchFromGitHub {
+    owner = "PySide";
+    repo = "PySide";
+    rev = version;
+    sha256 = "sha256-1s62HJriWWEik0maLqQP1EQeYzrQ6icYPRaYqShqGj0=";
```

Comment:
```
error: hash mismatch in fixed-output derivation '/nix/store/qbhnd734l5a24a7kpqqn2r6k3l5nj0hr-source.drv':
         specified: sha256-1s62HJriWWEik0maLqQP1EQeYzrQ6icYPRaYqShqGj0=
            got:    sha256-14XbihJRMk9WaeK6NUBV/4OMFZF8EBIJgEJEaCU8Ecg=
```

Reply:
i checked it again and got the same results `sha256-1s62HJriWWEik0maLqQP1EQeYzrQ6icYPRaYqShqGj0=`.

Reply:
```
source> trying https://github.com/PySide/PySide/archive/1.2.4.tar.gz
source>   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
source>                                  Dload  Upload   Total   Spent    Left  Speed
source> 100   120  100   120    0     0    111      0  0:00:01  0:00:01 --:--:--   111
source> 100 3813k    0 3813k    0     0   194k      0 --:--:--  0:00:19 --:--:--  170k
source> unpacking source archive /build/1.2.4.tar.gz
error: hash mismatch in fixed-output derivation '/nix/store/8nyp204iy5pj88kjx371l3qbwakmppx7-source.drv':
         specified: sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=
            got:    sha256-14XbihJRMk9WaeK6NUBV/4OMFZF8EBIJgEJEaCU8Ecg=
error: 1 dependencies of derivation '/nix/store/ihyf5hpmjskcffqr7d0shayxz2zhqfjw-python3.9-pyside-1.2.4.drv' failed to build
```

Reply:
most likely, it's a stale store path. FODs don't verify that they are still correct, if finds an FOD with the same hash, it will just re-use it.

change the hash sliglhtly, and you should see the new one